mmcv/cnn/bricks/transformer.py

```diff
@@ -55,6 +59,332 @@ def build_transformer_layer_sequence(cfg, default_args=None):
     return build_from_cfg(cfg, TRANSFORMER_LAYER_SEQUENCE, default_args)
 
 
+class AdaptivePadding(nn.Module):
+    """Applies padding to input (if needed) so that input can get fully covered
```

Comment:
The summary of this module is too long. Only keep one sentence as summary and leave others as description.